src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -368,12 +368,26 @@ const MatrixHeader = ({ context, showProjects, project }, reactContext) => {
 
     const rowCount = matrix => matrix.y['biosample_ontology.classification'].buckets.reduce((currentTotalCount, termName) => currentTotalCount + termName['biosample_ontology.term_name'].buckets.length, 0);
 
+    const query = new QueryString(context.search_base);
+    let organism = '';
+    if (query.getKeyValues('replicates.library.biosample.donor.organism.scientific_name').includes('Mus musculus')) {
+        organism = 'mouse';
+    } else if (query.getKeyValues('replicates.library.biosample.donor.organism.scientific_name').includes('Homo sapiens')) {
+        organism = 'human';
+    }
+    const matrixDescription = organism ?
+        <span>Project data from {organism} tissue, cell line, primary cell, and in vitro differentiated cell biosamples organized as reference epigenomes following guidelines set out by IHEC.</span>
```

Comment:
Look at the reference epigenome page. The radio button on the far left are displaced. A similar issue occurs on the ENCORE page. I do not think that is intended.

Reply:
That is intended. As I said this is interim, and I think it’s likely once we get all these matrix headers back to consistency, we’ll keep the title, internal tags badge, and the description all in one place at the top, and all controls by the matrix itself, instead of mixing controls and titles.